workflow/README.md

```diff
@@ -0,0 +1,47 @@
+# TPS Pipeline
+
+**TPS workflow overview**
+
+![Alt Text](https://github.com/ajshedivy/tps/blob/visualization_v2/workflow/TPS_workflow.png)
+
+The TPS/workflow module utilizes the TPS repository structure in order to run the TPS software end-to-end in order to produce Cytoscape sessions. 
+
+## Documentation
+
+- `TPS.sh`
+  - Runs TPS software 
+  - sets up and initializes workspace environment `py2cyto_envirnment`
+  - calls driver script `main.py`
+
+- `main.py`
+  - redirects TPS output files
+  - calls utility functions for processing network annotations
+  - calls visualization utilities
+  
+- `visualization_utilities.py`
+  - utility functions used for creating cyREST client 
+  - import ooutput groah files, style and annotation into Cytoscape
+  - TODO : search machine for Cytoscape executable
+  
+- `table_refactor.py`
```

Comment:
Do we still need separate refactoring code?  Can this eventually be merged with `table_generation.py`?  I would be okay keeping them separate initially, creating a test case, and then combining them later so that we can use the test case to ensure the combined version still works.

Reply:
yes, this would be possible. Right now the refactoring is done after the annotations are already generated. This can be merged into the actual generation of the annotations, although  I found it easier to edit them after the file is generated.

Reply:
Okay, let's make a new issue for merging these two Python scripts.  Then we can keep them in two separate stages for now, generate a test case, and use the test case to confirm combining them worked correctly.

Reply:
Noted in https://github.com/ajshedivy/tps/issues/18